x-pack/plugin/apm-data/src/main/resources/component-templates/metrics-apm@settings.yaml

```diff
@@ -12,3 +12,4 @@ template:
       # and this combination is incompatible with the
       # aggregate_metric_double field type.
       ignore_malformed: false
+      source.mode: synthetic
```

Comment:
I think we have a pre-existing issue in the template: there's no `index` before `mapping` :man_facepalming:

IIANM it doesn't matter for `ignore_malformed` since https://github.com/elastic/elasticsearch/pull/108746 was merged.

If that's correct, would you mind removing the bits above and fixing the nesting?

Reply:
I pushed: 9e40cb28cdc5950fe751937ed0c3101052927a3d

Reply:
Thank you!